jwst/photom/photom.py

```diff
@@ -906,6 +943,65 @@ def create_2d_conversion(self, model, exptype, conversion, waves, relresps,
 
         return conversion, no_cal
 
+    def create_1d_conversion(self, model, conversion, waves, relresps):
+        """Is this function a good idea?
```

Comment:
It seemed like a one-off addition, so I was thinking to add the function's code directly to the SOSS portion of photom_io rather than follow the 2d-equiv. structure. But it's done now!

Reply:
I like the more modular nature of the function, even if it's only called once. That way the higher-level routine doesn't get so cluttered up with detailed computations.